isce2_topsapp/__main__.py

```diff
@@ -82,6 +82,10 @@ def ensure_earthdata_credentials(username: Optional[str] = None, password: Optio
         )
 
 
+def string_is_true(s: str) -> bool:
+    return s.lower() == 'true'
+
```

Comment:
Should we add a check that `s.lower()` is in `['true', 'false']` so there are no unexpected results?

Reply:
@cmarshak you could add something like:
```python
if s.lower() is not in ('true', 'false'):
    raise argparse.ArgumentError('Only the strings `true` or `false` (any capitalization) may be provided.')
```
if you wanted it to crash if a nonsensical string was provided. We don't for a strictly HyP3 API, but for user-run stuff it's probably worth while.

Reply:
Should add a test or two at that point though

Reply:
:thinking: probably the right thing to do, so I'll do that